### PR TITLE
Update docker images for PHP 7.1+

### PR DESCRIPTION
**Pull request by juliushaertl**
This should fix the CI again, since we require PHP >=7.1 on master.

### CODE REVIEW

**Comment by danielkesselberg** [comment]
https://github.com/nextcloud/firstrunwizard/blob/b6df50eada48421ea415b0964981c473567d0ff9/tests/bootstrap.php#L36-L38

`PHPUnit_Framework_TestCase` => `PHPUnit\Framework\TestCase`. I'm not sure if we need this class_exists at all. PHPUnit uses composer to resolve classes for a long time. Could not find a `autoload.php` there.

**Comment by juliushaertl** [comment]
@danielkesselberg You're right, fixed :wink:

File: tests/Notification/NotifierTest.php [hunk 1]
```diff
@@ -208,7 +207,14 @@ public function testPrepare($language, $user, $changeName, $changeAvatar, $name,
...
 
-		$return = $this->notifier->prepare($notification, $language);
-		$this->assertEquals($notification, $return);
+		try {
+			$return = $this->notifier->prepare($notification, $language);
+			$this->assertEquals($notification, $return);
+		} catch (\Exception $exception) {
+			if ($subjectContains === false) {
+				$this->assertInstanceOf(InvalidArgumentException::class, $exception);
+			}
```
Comment (MorrisJobke) [inline tests/Notification/NotifierTest.php]: Shouldn't the else branch throw the exception again?

Comment (juliushaertl) [reply tests/Notification/NotifierTest.php]: Yes, good catch. Fixed.

**Pull request merged**